ordered_model/admin.py

```diff
@@ -88,7 +88,7 @@ def _get_model_info(self):
         }
 
 
-class OrderedTabularInline(admin.TabularInline):
+class OrderedInlineMixin:
```

Comment:
Should be `class OrderedInlineMixin(object):`, I think...

Reply:
Yes you are correct. I will change it.